feature-libs/asm/styles/components/_customer-selection.component.scss

```diff
@@ -145,7 +153,11 @@
       }
 
       @include forVersion(6.1) {
-        a {
+        .createNewCustomerLabel {
```

Comment:
try to use business unrelated class name, for example `xxx-weighted-label`, so that class can be reused for other scenarios.
or,
if you are sure the style is by purpose designed for one dedicated element, then use `id` instead, which is better on performance and give more clear semantic.
`class` usually designed for reuse.

Reply:
autofocus has been difined here: https://github.com/SAP/spartacus/pull/17376#discussion_r1191885328,  so for me, I have to use 'autofocus'.

changed 'createNewCustomerLabel'  to 'linkStyleLabel'